packages/wp-now/src/download.ts

```diff
@@ -54,19 +59,28 @@ async function downloadFileAndUnzip({
 					entry.pipe(fs.createWriteStream(filePath));
 				}
 			})
-			.promise();
+      .promise();
+    return true
 	} catch (err) {
 		console.error(`Error downloading or unzipping ${itemName}:`, err);
-	}
+  }
+  return false
 }
 
-export async function downloadWordPress(fileName = 'latest') {
-	await downloadFileAndUnzip({
-		url: WP_DOWNLOAD_URL,
-		destinationFolder: path.join(WORDPRESS_VERSIONS_PATH, fileName),
-		checkFinalPath: path.join(WORDPRESS_VERSIONS_PATH, fileName),
-		itemName: `WordPress ${fileName}`,
-	});
+export async function downloadWordPress(wordPressVersion = DEFAULT_WORDPRESS_VERSION) {
+  const finalFolder = path.join(WORDPRESS_VERSIONS_PATH, wordPressVersion);
+  const temporalFolder = os.tmpdir();
```

Comment:
this is totally fine, but also seems a bit non-standard. I'm used to seeing `tempFolder` or `temporaryFolder`, but not `temporalFolder`.

doesn't need to be changed, but it did stand out to me

Reply:
Renamed https://github.com/WordPress/wordpress-playground/pull/252/commits/405bcd28f86bd01a195f57d0959bd57be86e4270